dom.bs

```diff
@@ -2203,14 +2207,28 @@ for a given <a>slot</a> <var>slot</var>, run these steps:</p>
 <ol>
  <li><p>Let <var>result</var> be an empty list.</p></li>
 
- <li><p>If <var>slot</var>'s <a for=tree>root</a> is not a <a for=/>shadow root</a>, then return
+ <li><p>Let <var>root</var> be <var>slot</var>'s <a for=tree>root</a>.</p></li>
+
+ <li><p>If <var>root</var> is not a <a for=/>shadow root</a>, then return
  <var>result</var>.</p></li>
 
- <li><p>Let <var>host</var> be <var>slot</var>'s <a for=tree>root</a>'s
- <a for=DocumentFragment>host</a>.</p></li>
+ <li><p>Let <var>host</var> be <var>root</var>'s <a for=DocumentFragment>host</a>.</p></li>
 
  <li>
-  <p>For each <a>slottable</a> <a for=tree>child</a> of <var>host</var>, <var>slottable</var>, in
+  <p>If <var>root</var>'s <a for=ShadowRoot>slot assignment</a> is "<code>manual</code>",
+  then:</p>
+
+  <ol>
+   <li><p>Let <var>result</var> be an empty list.</p></li>
```

Comment:
This probably also needs to check that this slot is the first slot of a given name / default slot.

Reply:
I think this is ok also, for the same reasons as above.